code/modules/surgery/organs/eyes.dm

```diff
@@ -71,6 +71,7 @@
 /obj/item/organ/eyes/night_vision/zombie
 	name = "undead eyes"
 	desc = "Somewhat counterintuitively, these half-rotten eyes actually have superior vision to those of a living human."
+	sight_flags = SEE_MOBS
```

Comment:
This affects all zombe types. I.e mobs as well

Reply:
Only romerol zombies have those eyes.
datum/species/zombie/infectious are the only zombies that have the line `	mutanteyes = /obj/item/organ/eyes/night_vision/zombie`. The base zombie species, does not have that line.

Reply:
Alright